### PR TITLE
Home cards section

### LINKED ISSUE #31: home cards section

### PR DESCRIPTION
**Pull request by basil-kawak**

### CODE REVIEW

**Review by gbilgedal — APPROVED** [review]
looks great but next time, can you get rid of the empty lines like
->RelatedCards.jsx ln27-28
->index.css ln4-7

File: package.json [hunk 1]
```diff
@@ -6,10 +6,12 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "bootstrap": "^4.5.0",
```
Comment (louisrli) [inline package.json:9]: you should not be usıng both bootstrap and react-bootstrap. we should only be using react-bootstrap

`yarn remove bootstrap`

File: src/components/home/index.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* cards style */
```
Comment (louisrli) [inline src/components/home/index.css]: delete comment

File: src/components/home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import RelatedCards from './sections/RelatedCards';
+import './index.css';
+
+const Home = () => {
+    return (
+        <div className="Home">
```
Comment (louisrli) [inline src/components/home/Home.jsx]: run prettier please, indentation is wrong

Comment (louisrli) [inline src/components/home/Home.jsx]: all classnames should be camelcase

File: src/components/home/index.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+.cardsContainer {
```
Comment (louisrli) [inline src/components/home/index.css]: i think you want a more specific classname here. what if your teammate also has something that has cards in it? then both of them are called `.cardsContainer`, no one knows which card it refers to.

File: src/components/home/index.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    background-color: rgb(221, 225, 235);
+  
+    padding: 10% 100px 10% 100px;
```
Comment (louisrli) [inline src/components/home/index.css]: can you include a screenshot of what you're making please? both the design and what it looks like

File: src/components/home/index.css [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  .card {
+    transition: 0.5 ease;
+    cursor: pointer;
+    color: gray;
+  }
+  
+  .card:hover {
+    transform: scale(1.1);
+    box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.6);
+  }/* cards style ends */
```
Comment (louisrli) [inline src/components/home/index.css]: remove comment

File: src/components/home/sections/RelatedCards.jsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+                <Card>
+                    <Card.Img variant="top" src="holder.js/100px160" />
+                    <Card.Body>
+                    <Card.Text>
+                        This is a wider card with supporting text below as a natural lead-in to
+                        additional content. This content is a little bit longer.
+                    </Card.Text>
+                    </Card.Body>
+                    <Button variant="outline-light">Light</Button>{' '}
```
Comment (louisrli) [inline src/components/home/sections/RelatedCards.jsx]: delete space at end of line here

Comment (louisrli) [reply src/components/home/sections/RelatedCards.jsx]: and similar spaces elsewhere in this file

File: src/containers/App.jsx [hunk 8]
```diff
@@ -2,13 +2,17 @@ import React from 'react';
 import './index.css';
 import LangDropdown from "../components/LangDropdown"
 import { useTranslation } from "react-i18next";
+//components
```
Comment (louisrli) [inline src/containers/App.jsx]: remove comment

File: src/index.js [hunk 9]
```diff
@@ -4,6 +4,7 @@ import './index.css';
 import App from './containers/App';
 import * as serviceWorker from './serviceWorker';
 import "./i18n";
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (louisrli) [inline src/index.js:7]: should not be needed, use react-bootstrap only

**Comment by louisrli** [comment]
you can ignore my comments about yarn remove bootstrap, you should keep the package but make sure to use the react bootstrap components

File: src/components/home/index.css (hunk 5 above)
Comment (basil-kawak) [reply src/components/home/index.css]: this is what I've done.
my work
this is the design
design

Comment (basil-kawak) [reply src/components/home/index.css]: @louisrli here are the screenshots

**Comment by basil-kawak** [comment]
@louisrli I've modified the files as you told me to do and I think they meet the review requirement now, so can you please review them again!

File: src/components/home/index.css [hunk 10]
```diff
@@ -0,0 +1,34 @@
+.relatedCardsContainer {
+  background-color: #c8ccd6;
+  padding: 5rem 10% 5rem 10%;
+}
+
+.card {
```
Comment (louisrli) [inline src/components/home/index.css]: the problem with doing `.card` here is that if any of your teammates want to use `.card` (the bootstrap class, i think), then they will also inherit these styles, so you need to add a more specific class to the card that you're using

File: src/components/home/index.css [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+  transform: scale(1.1);
+  box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.6);
+}
+
+.card-img-top {
+  width: auto !important;
+  margin-bottom: 20px;
+}
+
+#scondAndThirdCardLogo {
```
Comment (louisrli) [inline src/components/home/index.css]: in general we never use IDs, make this a class. and fix typo, and maybe just call it cardLogo

File: src/components/home/sections/RelatedCards.jsx [hunk 12]
```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { CardDeck, Card } from "react-bootstrap";
+import logo1 from "../related-cards-logos/Logo1.png";
```
Comment (louisrli) [inline src/components/home/sections/RelatedCards.jsx]: let's name these files lowercase and starting with 0 index, which is more conventional

`logo0.png, logo1.png`, etc.

Comment (louisrli) [reply src/components/home/sections/RelatedCards.jsx]: i think you can also rename the directory `/home/logos/`

File: src/containers/App.jsx [hunk 13]
```diff
@@ -1,14 +1,16 @@
...
 import { useTranslation } from "react-i18next";
+import Home from "../components/home/Home";
 
 function App() {
   const { t, i18n } = useTranslation();
   return (
     <div className="App">
-      {t('Welcome.1')}
-      <LangDropdown/>
+      {t("Welcome.1")}
```
Comment (louisrli) [inline src/containers/App.jsx]: this should be lowercase actually, i know it was from a previous change but it should be `welcome.message` or something -- `1` is not a useful name

**Comment by basil-kawak** [comment]
@louisrli can you please have a look at the new changes!

**Comment by louisrli** [comment]
overall nice work. but the checks are failing. once you can fix the checks i will approve

**Pull request merged**